webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -273,39 +280,39 @@ class PermissionsResponderV1 extends Responder {
       *
       * @param projectIri the IRI of the project.
       * @param groups     the list of groups for which administrative permissions are retrieved and combined.
-      * @return a set of [[PermissionV1]].
+      * @return a set of [[PermissionADM]].
       */
-    private def administrativePermissionForGroupsGetV1(projectIri: IRI, groups: Seq[IRI]): Future[Set[PermissionV1]] = {
+    private def administrativePermissionForGroupsGeADM(projectIri: IRI, groups: Seq[IRI]): Future[Set[PermissionADM]] = {
 
         /* Get administrative permissions for each group and combine them */
-        val gpf: Seq[Future[Seq[PermissionV1]]] = for {
+        val gpf: Seq[Future[Seq[PermissionADM]]] = for {
             groupIri <- groups
-            //_ = log.debug(s"administrativePermissionForGroupsGetV1 - projectIri: $projectIri, groupIri: $groupIri")
+            //_ = log.debug(s"administrativePermissionForGroupsGetADM - projectIri: $projectIri, groupIri: $groupIri")
 
-            groupPermissions: Future[Seq[PermissionV1]] = administrativePermissionForProjectGroupGetV1(projectIri, groupIri).map {
-                case Some(ap: AdministrativePermissionV1) => ap.hasPermissions.toSeq
-                case None => Seq.empty[PermissionV1]
+            groupPermissions: Future[Seq[PermissionADM]] = administrativePermissionForProjectGroupGetADM(projectIri, groupIri, requestingUser = KnoraSystemInstances.Users.SystemUser).map {
+                case Some(ap: AdministrativePermissionADM) => ap.hasPermissions.toSeq
+                case None => Seq.empty[PermissionADM]
             }
 
         } yield groupPermissions
 
-        val allPermissionsFuture: Future[Seq[Seq[PermissionV1]]] = Future.sequence(gpf)
+        val allPermissionsFuture: Future[Seq[Seq[PermissionADM]]] = Future.sequence(gpf)
 
         /* combines all permissions for each group and removes duplicates  */
-        val result: Future[Set[PermissionV1]] = for {
-            allPermissions: Seq[Seq[PermissionV1]] <- allPermissionsFuture
+        val result: Future[Set[PermissionADM]] = for {
+            allPermissions: Seq[Seq[PermissionADM]] <- allPermissionsFuture
 
             // remove instances with empty PermissionV1 sets
-            cleanedAllPermissions: Seq[Seq[PermissionV1]] = allPermissions.filter(_.nonEmpty)
+            cleanedAllPermissions: Seq[Seq[PermissionADM]] = allPermissions.filter(_.nonEmpty)
 
             /* Combine permission sequences */
-            combined = cleanedAllPermissions.foldLeft(Seq.empty[PermissionV1]) { (acc, seq) =>
+            combined = cleanedAllPermissions.foldLeft(Seq.empty[PermissionADM]) { (acc, seq) =>
                 acc ++ seq
             }
             /* Remove possible duplicate permissions */
-            result: Set[PermissionV1] = PermissionUtilV1.removeDuplicatePermissions(combined)
+            result: Set[PermissionADM] = PermissionUtilV1.removeDuplicatePermissions(combined)
```

Comment:
Should `PermissionUtilV1` become `PermissionUtilADM`?

Reply:
yes, probably, since there is going to be only one version of permission handling implemented.

Reply:
renamed